core/src/main/resources/defaults/blueprints/cdp-sdx-medium-ha.bp

```diff
@@ -0,0 +1,428 @@
+{
+  "tags": {
+    "shared_services_ready": true
+  },
+  "description": "CDP 1.0 SDX-HA template with Atlas, HMS, Ranger and other services they are dependent on. Services like HDFS, HBASE have HA",
+  "blueprint": {
+    "cdhVersion": "7.0.0",
+    "displayName": "ha-datalake",
+    "hostTemplates": [
+      {
+        "refName": "master",
+        "roleConfigGroupsRefNames": [
+          "hbase-REGIONSERVER-BASE",
+          "hbase-MASTER-BASE",
+          "hdfs-DATANODE-BASE",
+          "hdfs-FAILOVERCONTROLLER-BASE",
+          "hdfs-JOURNALNODE-BASE",
+          "hdfs-NAMENODE-1",
+          "hive-HIVEMETASTORE-BASE",
+          "solr-SOLR_SERVER-BASE",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "master2",
+        "roleConfigGroupsRefNames": [
+          "hdfs-BALANCER-BASE",
+          "hbase-REGIONSERVER-BASE",
+          "hbase-MASTER-BASE",
+          "hdfs-DATANODE-BASE",
+          "hdfs-FAILOVERCONTROLLER-BASE",
+          "hdfs-JOURNALNODE-BASE",
+          "hdfs-NAMENODE-2",
+          "zookeeper-SERVER-BASE"
+        ]
+      },
+      {
+        "refName": "alpha",
+        "roleConfigGroupsRefNames": [
+          "ranger-RANGER_ADMIN-BASE",
+          "ranger-RANGER_USERSYNC-BASE",
+          "atlas-ATLAS_SERVER-1",
+          "hdfs-DATANODE-BASE",
+          "knox-KNOX_GATEWAY-BASE",
+          "ranger-RANGER_TAGSYNC-BASE",
+          "kafka-KAFKA_BROKER-BASE",
+          "zookeeper-SERVER-BASE",
+          "hdfs-JOURNALNODE-BASE"
+        ]
+      },
+      {
+        "refName": "alpha2",
+        "roleConfigGroupsRefNames": [
+          "kafka-GATEWAY-BASE",
+          "atlas-ATLAS_SERVER-BASE",
+          "kafka-KAFKA_BROKER-BASE",
```

Comment:
We think so. In case SDX cluster kafka is used to sync information between altas, ranger and HMS. 
We have start using and see and if it good enough. If not, we can increase it.